sdk/core/src/main/java/com/google/cloud/trace/util/TraceContext.java

```diff
@@ -75,6 +75,23 @@ public String toString() {
   }
 
   /**
+   * Transforms this context into an HTTP request header.
+   *
+   * @return this context as a header.
+   */
+  public String toHeader() {
+    StringBuilder builder =
+        new StringBuilder()
+            .append(traceId.getTraceId().toString(16))
```

Comment:
The trace id string should be 32 characters long, padded with zeroes, if necessary, or the API may reject it.